### PR TITLE
Adding descriptions

### PR DESCRIPTION
**Pull request by get-itips**
Closes https://github.com/MicrosoftDocs/office-docs-powershell/issues/6922

I am adding also the description for DisableAppViews, but it seems it is not working so well, can you test in your tenants @dariomws @tseward ?

Thanks

### CODE REVIEW

File: sharepoint/sharepoint-ps/sharepoint-online/Set-SPOSite.md [hunk 1]
```diff
@@ -570,8 +574,11 @@ Accept wildcard characters: False
 ```
 
 ### -DisableCompanyWideSharingLinks
+Disables People in your organization links. For more information, see [People in your organization sharing links](https://docs.microsoft.com/en-us/microsoft-365/solutions/microsoft-365-limit-sharing?view=o365-worldwide#people-in-your-organization-sharing-links).
```
Comment (dariomws) [inline sharepoint/sharepoint-ps/sharepoint-online/Set-SPOSite.md:577]:
```suggestion
Disables People in your organization links. For more information, see [People in your organization sharing links](https://docs.microsoft.com/microsoft-365/solutions/microsoft-365-limit-sharing#people-in-your-organization-sharing-links).
```

File: sharepoint/sharepoint-ps/sharepoint-online/Set-SPOSite.md [hunk 2]
```diff
@@ -570,8 +574,11 @@ Accept wildcard characters: False
...
+Possible values
 
-PARAMVALUE: Unknown | Disabled | NotDisabled
+- Disabled
+- NotDisabled
```
Comment (dariomws) [inline sharepoint/sharepoint-ps/sharepoint-online/Set-SPOSite.md:581]: Unknow is available. Maybe we should leave it.

File: sharepoint/sharepoint-ps/sharepoint-online/Set-SPOSite.md [hunk 3]
```diff
@@ -555,7 +555,11 @@ Accept wildcard characters: False
 
 ### -DisableAppViews
 
-PARAMVALUE: Unknown | Disabled | NotDisabled
+Disables the Power Apps button.
+Possible values
```
Comment (dariomws) [inline sharepoint/sharepoint-ps/sharepoint-online/Set-SPOSite.md:559]:
```suggestion
Possible values:
```

**Review by dariomws — CHANGES_REQUESTED** [review]
I tested them, and DisableFlows worked, not DisableAppViews.

Before:
image
image

After:
image
image

I'll check again in some hours...

**Comment by tseward** [comment]
The description/intention of the switch is correct. If it isn't working, that's another issue to raise.

**Review by tseward — APPROVED** [review]

File: sharepoint/sharepoint-ps/sharepoint-online/Set-SPOSite.md (hunk 2 above)
Comment (get-itips) [reply sharepoint/sharepoint-ps/sharepoint-online/Set-SPOSite.md:581]: yes, but it not "settable", what should we do? unknown is a state only when you have never set this.

Comment (dariomws) [reply sharepoint/sharepoint-ps/sharepoint-online/Set-SPOSite.md:581]: Good question... not sure if it's better to leave it and add a comment in brackets like "not settable" or what you propose. @tseward?

**Review by dariomws — APPROVED** [review]

File: sharepoint/sharepoint-ps/sharepoint-online/Set-SPOSite.md (hunk 2 above)
Comment (tseward) [reply sharepoint/sharepoint-ps/sharepoint-online/Set-SPOSite.md:581]: I agree with a not settable type verbiage.

**Comment by yogkumgit** [comment]
@Kateyanne Please merge this PR. Thanks.

**Pull request merged**